app/services/support_interface/external_report_exports/applications_by_status_export.rb

```diff
@@ -0,0 +1,12 @@
+module SupportInterface
```

Comment:
The monthly stats code does not currently live in the 'support' namespace as it is a publicly accessible. At the time it did not feel like its home should be within the candidate or support namespaces ... 
Oh ... but I guess the exports are only made available via the support UI?

Reply:
Yea the exports will live in the Support UI. It's a good question about where the dashboard should sit actually 🤔